<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Network_Voip</class>
 <widget class="QMainWindow" name="Network_Voip">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network Escalation Ticket - VOIP / Nextiva</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="3">
     <widget class="QTextEdit" name="ticketTemplate">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>370</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="3">
        <widget class="QPushButton" name="Clear">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="Copy">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Copy to Clipboard</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Generate Ticket</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>370</width>
        <height>610</height>
       </size>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>368</width>
         <height>608</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="ticketInfo">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; text-decoration: underline;&quot;&gt;General Information:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="group_Current_Customer">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Current Customer</string>
          </property>
          <layout class="QFormLayout" name="formLayout_5">
           <item row="0" column="0">
            <widget class="QGroupBox" name="g_plugged">
             <property name="title">
              <string>Is the VOIP Adapter Plugged in First?</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <layout class="QFormLayout" name="formLayout_4">
              <item row="0" column="0">
               <widget class="QRadioButton" name="yes_Plugged">
                <property name="text">
                 <string>Yes</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QRadioButton" name="no_Plugged">
                <property name="text">
                 <string>No</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="g_registered">
             <property name="title">
              <string>Is the VOIP Line in a Registered State?</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <layout class="QFormLayout" name="formLayout_6">
              <item row="1" column="0">
               <widget class="QRadioButton" name="yes_registered">
                <property name="text">
                 <string>Yes</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QRadioButton" name="no_registered">
                <property name="text">
                 <string>No</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QGroupBox" name="g_line1">
             <property name="title">
              <string>Is the Phone Plugged Into Line 1?</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <layout class="QFormLayout" name="formLayout_7">
              <item row="0" column="0">
               <widget class="QRadioButton" name="yes_line1">
                <property name="text">
                 <string>Yes</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QRadioButton" name="no_line1">
                <property name="text">
                 <string>No</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QRadioButton" name="other_line1">
                <property name="text">
                 <string>Other</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="other_line1Text">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QGroupBox" name="group_New_Customer">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>New Customer</string>
          </property>
          <layout class="QFormLayout" name="formLayout_8">
           <item row="0" column="0">
            <widget class="QGroupBox" name="g_paying">
             <property name="title">
              <string>Is the Customer Currently Paying for VOIP?</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <layout class="QFormLayout" name="formLayout_9">
              <item row="0" column="0">
               <widget class="QRadioButton" name="yes_paying">
                <property name="text">
                 <string>Yes</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QRadioButton" name="no_paying">
                <property name="text">
                 <string>No</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="g_assignment">
             <property name="title">
              <string>Number Assignment:</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <layout class="QFormLayout" name="formLayout_10">
              <item row="0" column="0">
               <widget class="QRadioButton" name="new_assignment">
                <property name="text">
                 <string>New</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QRadioButton" name="ported_assignment">
                <property name="text">
                 <string>Ported</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="cstEmail">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="placeholderText">
                 <string>Correct Email Address</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_callerID">
             <property name="text">
              <string>Caller ID Name to Show on Outbound Calls:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="cstCallerID">
             <property name="text">
              <string/>
             </property>
             <property name="maxLength">
              <number>15</number>
             </property>
             <property name="placeholderText">
              <string>Maxium 15 characters</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>370</width>
        <height>610</height>
       </size>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>351</width>
         <height>702</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; text-decoration: underline;&quot;&gt;Customer Information:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_cstName">
          <property name="text">
           <string>Customer Name:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLineEdit" name="cstName"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_cstID">
          <property name="text">
           <string>Customer ID:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLineEdit" name="cstID"/>
        </item>
        <item row="5" column="0">
         <widget class="QGroupBox" name="cst_Group_Status">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Customer Status:</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QRadioButton" name="cstCurrent">
             <property name="text">
              <string>Current Customer</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="cstNew">
             <property name="text">
              <string>New Customer</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="20" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="21" column="0">
         <widget class="Line" name="line_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>0</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="22" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; text-decoration: underline;&quot;&gt;Ticket Information:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="23" column="0">
         <widget class="QLabel" name="label_QFirmware">
          <property name="text">
           <string>Firmware / Config File Updated in Adapter</string>
          </property>
         </widget>
        </item>
        <item row="24" column="0">
         <widget class="QCheckBox" name="tktQFirmware">
          <property name="text">
           <string>Completed.</string>
          </property>
         </widget>
        </item>
        <item row="25" column="0">
         <widget class="QLabel" name="label_mac">
          <property name="text">
           <string>MAC Address of VOIP Adapter:</string>
          </property>
         </widget>
        </item>
        <item row="26" column="0">
         <widget class="QLineEdit" name="tktVoipMAC">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="27" column="0">
         <widget class="QLabel" name="label_ip">
          <property name="text">
           <string>Public or Private IP Address of Adapter:</string>
          </property>
         </widget>
        </item>
        <item row="28" column="0">
         <widget class="QLineEdit" name="tktIP"/>
        </item>
        <item row="29" column="0">
         <widget class="QLabel" name="label_reason">
          <property name="text">
           <string>Reason for Escalation:</string>
          </property>
         </widget>
        </item>
        <item row="30" column="0">
         <widget class="QTextEdit" name="tktPurpose">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>80</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <property name="tabChangesFocus">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>Why are you creating the escalation ticket?</string>
          </property>
         </widget>
        </item>
        <item row="52" column="0">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1140</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>scrollArea_2</tabstop>
  <tabstop>cstName</tabstop>
  <tabstop>cstID</tabstop>
  <tabstop>cstCurrent</tabstop>
  <tabstop>cstNew</tabstop>
  <tabstop>tktQFirmware</tabstop>
  <tabstop>tktVoipMAC</tabstop>
  <tabstop>tktIP</tabstop>
  <tabstop>tktPurpose</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>yes_Plugged</tabstop>
  <tabstop>no_Plugged</tabstop>
  <tabstop>yes_registered</tabstop>
  <tabstop>no_registered</tabstop>
  <tabstop>yes_line1</tabstop>
  <tabstop>no_line1</tabstop>
  <tabstop>other_line1</tabstop>
  <tabstop>other_line1Text</tabstop>
  <tabstop>yes_paying</tabstop>
  <tabstop>no_paying</tabstop>
  <tabstop>new_assignment</tabstop>
  <tabstop>ported_assignment</tabstop>
  <tabstop>cstEmail</tabstop>
  <tabstop>cstCallerID</tabstop>
  <tabstop>ticketTemplate</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>Copy</tabstop>
  <tabstop>Clear</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
